pyproject.toml

```diff
@@ -79,13 +79,13 @@ coloredlogs = ">=10,<16"
 sanic = "^21.12.0"
 typing-extensions = ">=4.1.1,<5.0.0"
 Sanic-Cors = "^2.0.0"
-prompt-toolkit = "^3.0,<3.0.29"
+prompt-toolkit = ">=3,<4"
 "ruamel.yaml" = ">=0.16.5,<0.18.0"
 websockets = ">=10.0,<12.0"
 pluggy = "^1.0.0"
 opentelemetry-api = "~1.15.0"
 opentelemetry-sdk = "~1.15.0"
-opentelemetry-exporter-jaeger = "~1.15.0"
```

Comment:
Two points here, if we're updating this dependency, all the other open telemetry dependencies have to be on the same minor. In addition, we have to align this dependency version with the identical dependencies in rasa-plus.

Reply:
Thanks @ancalita for pointing it out. As it seems to be much more work than expected, I would close this PR and create an issue for it. We are currently focusing on a project that needs to be done by end of the year.